addons/etcd/scripts/etcd-member-leave.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -ex
+endpoints=$(echo $KB_MEMBER_ADDRESSES | tr ',' '\n')
+leaverEndpoint=$(echo "$endpoints" | grep $KB_LEAVE_MEMBER_POD_NAME)
```

Comment:
should consider the situation that leaverEndpoint is empty

Reply:
fixed